src/views/tools/image_management.test.ts

```diff
@@ -13,11 +12,55 @@ import MemoryStoragePortal from "../../server/services/storage_portal/storage_po
 import { timeout } from "../../shared/utils";
 import { getAllImageInfos, startUpdatingImages } from "./image_management";
 import { ImageInfo } from "./types";
+import { spawn } from "child_process";
+import { Level, setLogLevel } from "../../server/log";
+
+// Helper to save a jpeg buffer to a file, and run imagemagick's identify tool on it.
+function identify(jpgData: Buffer): Promise<string> {
+  return new Promise((resolve) => {
+    // TODO: Remove dependency on the filesystem, see if there's a way to send the data
+    // through a stdin stream or something.
+    const tmpFile =
+      "/tmp/imagemanagement_img_" + Math.floor(Math.random() * 100) + ".jpg";
```

Comment:
What's the purpose of having Math.floor and Math.random? Why not overwrite a single file each time?

Reply:
It's to reduce the likelihood of two concurrently running tests conflicting with each other.  For example, you might run `npm test` at the same time Jenkins runs a test, and then you'd clobber each others' files.

There's still a 1% chance you'll land on the same file name, which is bad. But I think it's acceptable for now, because I plan on getting rid of the file system dependency all together soon.